_resources/port1.0/group/haskell_cabal-1.0.tcl

```diff
@@ -80,6 +80,13 @@ default build.cmd           {${haskell_cabal.bin}}
 default build.target        {new-build}
 default build.args          {${haskell_cabal.default_args}}
 
+# ensure /usr/bin/gcc is used; see https://github.com/haskell/cabal/issues/1325
```

Comment:
this comment is now incorrect, as the changes below no longer force the user of `/usr/bin/gcc` but whatever `${configure.cc}` is set up

Reply:
Fixed.